docs/developer/app-store/apps/emails-and-messages/overview.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Overview
+sidebar_position: 1
+---
+
+# Introduction
+
+Emails and messages (EAM) application is responsible for sending emails and messages to the customers. It is connected to the Saleor API via webhooks, which notify the application about the events. The messages are delivered by external providers configured in the application.
+
+## Suported providers
+
+Emails and Messages app currently supports the following providers:
+
+- [Sendgrid](sendgrid) - Create templates in the Sendgrid visual editor and send them to your customers.
+- [SMTP](smtp) - Use your own SMTP server to send emails. Email templates are created using MJML and handlebars.
+
+## Supported events
+
+### Account confirmation
+
+A message containing an account activation token after registration. To activate the account, your storefront should perform an API call according to [the documentation](../../../users#registration-with-email-confirmation).
+
+If you don't want to use the email confirmation, you can disable it in the Saleor dashboard:
+
+1. Open the Saleor dashboard
+1. Navigate to the Configuration page
+1. Click on the Site settings card
+1. Update the `User registration` form
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account password reset
+
+Message sent after requesting a password change. The payload contains a token that has to be attached to the request to the Saleor API to change the password. To implement a storefront interface for password change, follow [the documentation](../../../users#resetting-the-password).
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account password reset for accounts created by the staff
+
+As in the previous event, but for accounts created by Staff users in the Saleor Dashboard.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account email address change
+
+Message sent on the email address change request. Follow the [documentation](../../../users#changing-the-email-address) for more details.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account email change confirmation
+
+Confirmation is sent to the new address after following directions from the previous message.
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Account deletion request
+
+A message with a token that's used for account deletion. To implement a storefront interface for account deletion, follow [the documentation](../../../users#deleting-the-account).
+
+The event has been implemented using the `NOTIFY` webhook.
+
+### Invoice sent
+
+The event is triggered when the Staff users use the `Sent invoice` action. The message contains a link to the invoice.
+
+The event has been implemented using the `INVOICE_SENT` webhook.
+
+### Order has been confirmed
```

Comment:
**polish:**
I am suggesting a form used in the previous headings ("Account deletion"). If you agree, you can change the rest of the headings too.

```suggestion
### Order confirmation
```